site_root/input/resources/profiles/StructureDefinition-drs-attachment.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
Since `DocumentReference.content.attachment` is a required attribute, if we want to use `drs-attachment`, then we would need to use it as an extension of `DocumentReference.content`. Then, we would end up having two attachments. From the example resource, you've used the `drs-url` extension with `DocumentReference.content.attachment`, not instantiating `drs-attachment` for `DocumentReference.content`. Given this, I wonder if we still need `drs-attachment`?